evals/registry/completion_fns/langchain_llms.yaml

```diff
@@ -0,0 +1,20 @@
+langchain_gpt-3.5-turbo:
```

Comment:
Should we prefix with something more "special" like `langchain/`? Other completion_fns can follow the same format.

Reply:
yeah thats cool I can do that